src/core.c/allomorphs.pm6

```diff
@@ -189,26 +191,11 @@ multi sub infix:<cmp>(ComplexStr:D $a, RatStr:D     $b) { $a.Complex cmp $b.Rat
 multi sub infix:<cmp>(ComplexStr:D $a, NumStr:D     $b) { $a.Complex cmp $b.Num     || $a.Str cmp $b.Str }
 multi sub infix:<cmp>(ComplexStr:D $a, ComplexStr:D $b) { $a.Complex cmp $b.Complex || $a.Str cmp $b.Str }
 
-
-multi sub infix:<eqv>(IntStr:D     $a, IntStr:D     $b) { $a.Int     eqv $b.Int     && $a.Str eqv $b.Str }
-multi sub infix:<eqv>(IntStr:D     $a, RatStr:D     $b --> False) {}
-multi sub infix:<eqv>(IntStr:D     $a, NumStr:D     $b --> False) {}
-multi sub infix:<eqv>(IntStr:D     $a, ComplexStr:D $b --> False) {}
-
-multi sub infix:<eqv>(RatStr:D     $a, IntStr:D     $b --> False) {}
-multi sub infix:<eqv>(RatStr:D     $a, RatStr:D     $b) { $a.Rat     eqv $b.Rat     && $a.Str eqv $b.Str }
-multi sub infix:<eqv>(RatStr:D     $a, NumStr:D     $b --> False) {}
-multi sub infix:<eqv>(RatStr:D     $a, ComplexStr:D $b --> False) {}
-
-multi sub infix:<eqv>(NumStr:D     $a, IntStr:D     $b --> False) {}
-multi sub infix:<eqv>(NumStr:D     $a, RatStr:D     $b --> False) {}
-multi sub infix:<eqv>(NumStr:D     $a, NumStr:D     $b) { $a.Num     eqv $b.Num     && $a.Str eqv $b.Str }
-multi sub infix:<eqv>(NumStr:D     $a, ComplexStr:D $b --> False) {}
-
-multi sub infix:<eqv>(ComplexStr:D $a, IntStr:D     $b --> False) {}
-multi sub infix:<eqv>(ComplexStr:D $a, RatStr:D     $b --> False) {}
-multi sub infix:<eqv>(ComplexStr:D $a, NumStr:D     $b --> False) {}
-multi sub infix:<eqv>(ComplexStr:D $a, ComplexStr:D $b) { $a.Complex eqv $b.Complex && $a.Str eqv $b.Str }
+multi sub infix:<eqv>(Allomorph:D $a, Allomorph:D $b --> Bool:D) is default {
```

Comment:
I get why the `is default` here is needed; it makes me a little uneasy to see it, but I didn't think up an example where it gets us into bother.